WooCommerce/Classes/POS/ViewModels/TotalsViewModel.swift

```diff
@@ -21,6 +21,7 @@ final class TotalsViewModel: ObservableObject, TotalsViewModelProtocol {
     }
 
     @Published var cardPresentPaymentOnboardingViewModel: CardPresentPaymentsOnboardingViewModel?
+    @Published var cardPresentPaymentOnboardingURL: URL?
```

Comment:
I'm wondering if this is necessary 🤔 Maybe using `@State var` within the view for such cases would be enough? We don't have strict (or likely any) guidelines for this within SwiftUI. What do you think?

Reply:
Yup a state variable would be easier, but since the onboarding view is currently defined in `PointOfSaleDashboardView`, I was hoping not to add sub-level view detail to the top-level view whose property list is pretty clean now. When updating the sheet to POS modal in https://github.com/woocommerce/woocommerce-ios/issues/14209, I might create a wrapper for the onboarding view for POS and move the properties there.